tools/kf_mskcc_vcf2maf.cwl

```diff
@@ -0,0 +1,58 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: kf-mskcc-vcf2maf
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 4000
+    coresMin: 2
+  - class: DockerRequirement
+    dockerPull: 'migbro/vcf2maf:v1.0.1'
```

Comment:
Should this be from sbgenomics?

Reply:
Yes! I was planning on changing this once the dockerfile was approved. Chicken and the egg man....

Reply:
Updated docker and it ran successfully here: https://cavatica.sbgenomics.com/u/zhangb1/kf-somatic-tools-test/tasks/1e1ae16d-90e9-4932-93e5-98c6984dafeb/